db/schema.rb

```diff
@@ -70,8 +70,11 @@
     t.boolean  "visible",             limit: 1,   default: true
     t.string   "runner_type",         limit: 255, default: "box"
     t.boolean  "best_submit_results", limit: 1,   default: false
+    t.integer  "group_id",            limit: 4,   default: 1,     null: false
   end
 
+  add_index "contests", ["group_id"], name: "index_contests_on_group_id", using: :btree
```

Comment:
I don't see a foreign key definition in the schema. You have `foreign_key: true` in the migration and this should make a change in the schema. Are you sure you ran the latest version of the above migration against your local DB?